soccer/gameplay/plays/offense/clear.py

```diff
@@ -0,0 +1,95 @@
+import main
+import robocup
+import behavior
+import constants
+import enum
+import standard_play
+import evaluation
+import tactics.coordinated_pass
+import skills.move
+import skills.capture
+import random
+
+#Based on clear sitation
+#    Have ball in defense
+#   Want to Kick ball up field and get other robots in position
+#
+
+class Clear(standard_play.StandardPlay):
+
+    class State(enum.enum)
+        get_ball = 1, 'Get the ball and movee other robots up'
+        clear_ball = 2, 'Kick the ball upfield'
+
+    def __init__(self):
+        super().__init__(continuous = False)
+
+        for s in Clear.State:
+            self.add_state(s, behavior.Behavior.State.running)
+
+        #Points to move up offense bots to
+        self.offense_point_1 = robocup.Point(0.40*constants.Field.Width,
```

Comment:
What if we want different numbers of offense and midfield robots? These shouldn't be hardcoded but we should be able to alter it easily, especially if we want to move up to 11 robots eventually